zebra/zebra_vty.c

```diff
@@ -1793,6 +1793,21 @@ DEFUN (show_evpn_vni,
 	return CMD_SUCCESS;
 }
 
+DEFUN(show_evpn_vni_detail, show_evpn_vni_detail_cmd,
+      "show evpn vni detail [json]", SHOW_STR
+      "EVPN\n"
+      "VxLAN information\n"
```

Comment:
This should be `"VxLAN Network Identifier\n"` as is used in other places

Reply:
Done. Fixed at another place where I copied it from.